applications/nrf_desktop/README.rst

```diff
@@ -1311,7 +1311,7 @@ The HID over GATT profile specification requires Bluetooth Peripherals to define
 * HID Service - Handled in the :ref:`nrf_desktop_hids`.
 * Battery Service - Handled in the :ref:`nrf_desktop_bas`.
 * Device Information Service - Implemented in Zephyr and enabled with :option:`CONFIG_BT_GATT_DIS`.
-  It can be configured using Kconfig options with the ``CONFIG_BT_GATT_DIS`` prefix.
+  It can be configured using Kconfig options with the :option:`CONFIG_BT_GATT_DIS` prefix.
```

Comment:
@MarekPieta , can you check these options after rebase?

Reply:
This is actually used as a prefix here, so I would prefer not to make it an option.

Reply:
OK, then remove :option:.